src/payment/payment-methods.mock.ts

```diff
@@ -411,6 +412,13 @@ export function getPaymentMethodsMeta() {
     };
 }
 
+export function getPaymentMethodsWith3ds(): PaymentMethod[] {
+    const amazon3ds = getAmazonPay();
+    amazon3ds.config.is3dsEnabled = true;
```

Comment:
You can do this like this just for immutability, but it's okay as it is. 🍹 
```suggestion
    return [
        {
            ... amazon3ds,
            config: {
                ...amazon3ds.config,
                is3dsEnabled: true
            }
        }
    ];
```

Reply:
You know what? Ignore this one.